src/checkbox-group/middleware.tsx

```diff
@@ -39,7 +41,13 @@ export const checkboxGroup = factory(({ middleware: { icache } }) => {
 
 		return (key: string) => ({
 			checked(checked?: boolean) {
-				const values = icache.getOrSet('values', {});
+				const values =
```

Comment:
could this be done when it's checked to see if `value` is `undefined` rather than every time `checked` is called?